### PR TITLE
{chem}[intel/2016a] i-PI v1.0-20160213

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/i/i-PI/i-PI-1.0-20160213-intel-2016a-Python-2.7.11.eb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+easyblock = 'PythonPackage'
+
+name  ='i-PI'
+version = '1.0-20160213'
+commit = '2a09a6d'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/i-pi/i-pi'
+description = """A Python wrapper for (ab initio) (path integrals) molecular dynamics"""
```
Comment (wpoely86) [inline easybuild/easyconfigs/i/i-PI/i-PI-1.0-20160213-intel-2016a-Python-2.7.11.eb]: if it's a wrapper, doesn't need something to wrap?

Comment (boegel) [inline easybuild/easyconfigs/i/i-PI/i-PI-1.0-20160213-intel-2016a-Python-2.7.11.eb]: it wraps several things, but does so at runtime; including something specific as a dependency would artificially limit it to one particular things

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2159.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/8efce123ba950076957c00234cec2699 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2402.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/cc20444fb488b3ead3727a78293d0da5 for a full test report.

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**